readthedocs/wsgi.py

```diff
@@ -8,7 +8,7 @@
 # This application object is used by any WSGI server configured to use this
 # file. This includes Django's development server, if the WSGI_APPLICATION
 # setting points here.
-from django.core.wsgi import get_wsgi_application  # pylint: disable=wrong-import-position
+from django.core.wsgi import get_wsgi_application  # noqa
```

Comment:
I do generally prefer explicit disables, though I know that prospector was the one making this easier. For example, prospetor knows that pylint and flake8 (or whatever) throw an error on the same line, but pylint is ignoring -- it will therefore ignore the flake8 error as well. I don't think our pre-commit pass has anything comparable unfortunately.

``#noqa`` becomes a generic exception handler of sorts, where who knows what else we're ignoring when we use `#noqa`

Reply:
Not sure if this is possible. worth making an issue on common?

Reply:
I also like being explicit here. I didn't find how to avoid the checking that was failing, though. It was something related to the import not at the top of the file and it wasn't pylint (I think it was flake --don't remember). That's why I just added `# noqa`.